libvast/vast/concept/printable/vast/data.hpp

```diff
@@ -19,11 +19,14 @@
 #include "vast/concept/printable/vast/pattern.hpp"
 #include "vast/concept/printable/vast/subnet.hpp"
 #include "vast/concept/printable/vast/type.hpp"
-#include "vast/data.hpp"
 #include "vast/detail/escapers.hpp"
 #include "vast/detail/overload.hpp"
 #include "vast/detail/string.hpp"
 
+#ifndef VAST_PRINTABLE_DO_NOT_INCLUDE_DATA
```

Comment:
Whats the use-case for defining these macros?

Reply:
To avoid cyclic includes; the printable include includes data, and that caused some compile errors when trying to use printable below to print data. Long-term we want to get rid of printable anyways, so I figured this was a decent enough workaround.

Reply:
I'm confused, why isn't this already solved by `#pragma once`? Anyways, if this an internal workaround we should probably adjust the name to make it clear, currently it looks like a regular compile option.

Reply:
Hm. Just tried removing them and apparently that works. Not sure why, but without that my clangd crashed earlier. Maybe that was just an unrelated issue, who knows.